### PR TITLE
Feature/integrate datetime field update unit test

### PR DESCRIPTION
**Pull request by Sarin-Udompanish**
Update unit test.

### CODE REVIEW

File: packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js [hunk 1]
```diff
@@ -3,25 +3,48 @@ import { fixture, expect, elementUpdated } from '@refinitiv-ui/test-helpers';
...
+    });
+    it('DOM structure is correct when timepicker and with-seconds', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" timepicker with-seconds opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('DOM structure is correct when range timepicker', async () => {
+      const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" timepicker range opened></ef-datetime-picker>');
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+  });
```
Comment (goremikins) [inline packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js:64]: Add here a test case with formatOptions as well, to test date-only; time-only and date-time test scenarios.

File: packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js [hunk 2]
```diff
@@ -31,73 +54,23 @@ describe('datetime-picker/DatetimePicker', () => {
       expect(el.opened).to.be.equal(false);
       expect(el.error).to.be.equal(false);
       expect(el.warning).to.be.equal(false);
-      expect(el.inputTriggerDisabled).to.be.equal(false);
       expect(el.inputDisabled).to.be.equal(false);
       expect(el.popupDisabled).to.be.equal(false);
       expect(el.timepicker).to.be.equal(false);
       expect(el.duplex).to.be.equal(null);
       expect(el.readonly).to.be.equal(false);
       expect(el.disabled).to.be.equal(false);
```
Comment (goremikins) [inline packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js:88]: Add `formatOptions` and `locale`

File: packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js [hunk 3]
```diff
@@ -9,108 +9,78 @@ describe('datetime-picker/Value', () => {
...
+      expect(inputElement(el).value).to.be.equal('2020-04-01', 'Input value has not updated');
     });
     it('It should be possible to select value in range duplex mode', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" opened range duplex></ef-datetime-picker>');
       el.views = ['2020-04', '2020-05'];
       await elementUpdated(el);
-      await nextFrame();
-      await nextFrame();
+      await nextFrame(el);
+      await nextFrame(el);
```
Comment (goremikins) [inline packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js:53]: Here and in other places. I think these multiple `nextFrame` calls were added to accommodate IE11. Please double check if you still need them.

File: packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js [hunk 4]
```diff
@@ -9,108 +9,78 @@ describe('datetime-picker/Value', () => {
...
       expect(timePicker.seconds).to.equal(15);
     });
     it('It should be possible to change timepicker value', async () => {
-      const el = await fixture('<ef-datetime-picker lang="en-gb" opened timepicker with-seconds value="2020-04-21T13:14:15"></ef-datetime-picker>');
-      const timePicker = el.timepickerEl;
-      typeText(timePicker, '16:17:18');
+      const el = await fixture('<ef-datetime-picker lang="en-gb" opened timepicker show-seconds value="2020-04-21T13:14:15"></ef-datetime-picker>');
+      typeText(timePickerElement(el), '16:17:18');
       expect(el.value).to.equal('2020-04-21T16:17:18');
     });
```
Comment (goremikins) [inline packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js:82]: Add a test, `it should be possible to change formatOptions value` and `it should be possible to change locale value`

File: documents/src/pages/elements/datetime-picker.md [hunk 5]
```diff
@@ -66,51 +66,7 @@ datetimePicker.value = '2019-03-20';
```
Comment (goremikins) [inline documents/src/pages/elements/datetime-picker.md]: To keep moving, please create a separate PR with documentation changes, and leave this one just for tests and comments.

File: packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js (hunk 3 above)
Comment (Sarin-Udompanish) [reply packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js:53]: Updated. It still need to call nextFrame one time after updating the views.

File: packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js (hunk 4 above)
Comment (Sarin-Udompanish) [reply packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js:82]: Added.

File: packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js (hunk 1 above)
Comment (Sarin-Udompanish) [reply packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js:64]: Added.

**Review by goremikins — APPROVED** [review]

**Pull request merged**